spec/game_spec.rb

```diff
@@ -37,16 +37,22 @@
       expect(@game.robo_board.ships).to eq([@game.robo_cruiser, @game.robo_submarine])
     end
 
-    it 'human places ships randomly in valid locations' do
-    
+    it 'human places ships in valid locations' do
+      @game.human_place_ship(@game.human_cruiser)
+      @game.human_place_ship(@game.human_submarine)
+      @game.human_place_ship(@game.human_cruiser, ["A1", "A2", "A3"])
+      expect(@game.human_board.cells["A1"].ship).to be_a(Ship)
```

Comment:
I think I'd suggest testing every cell a ship is meant to be in and also testing that the same ship is in every cell here.

Reply:
Got it - I'll work on it.  Thanks!